plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/CompositeHandler.java

```diff
@@ -0,0 +1,56 @@
+/******************************************************************************* 
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.reddeer.core.handler;
+
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.jboss.reddeer.common.util.Display;
+import org.jboss.reddeer.common.util.ResultRunnable;
+
+/**
+ * Contains methods for handling UI operations on {@link Composite} widgets.
+ * 
+ * @author rawagner
+ *
+ */
+public class CompositeHandler {
```

Comment:
composite is children of control and control is children of widget in Eclipse. Could we reflect this fact in handlers hierarchy?